### PR TITLE
Fix "License" vs. "Licence" typos

### PR DESCRIPTION
**Pull request by reecebenson**

### CODE REVIEW

**Comment by frebib** [comment]
Thanks for the PR, but the spelling is intentionally correct: https://www.grammarly.com/blog/licence-license/
There is no excuse for our inconsistency though. I'll look at getting those resolved and try not to break compatibility.

**Comment by reecebenson** [comment]
> 
> 
> Thanks for the PR, but the spelling is intentionally correct: https://www.grammarly.com/blog/licence-license/
> There is no excuse for our inconsistency though. I'll look at getting those resolved and try not to break compatibility.

Hi, no worries. The main change here was ensuring that the docs for README were consistent. Spent an embarrassing amount of time setting `LICENSE_KEY`, when I should've been setting `LICENCE_KEY`!

**Comment by reecebenson** [comment]
@frebib I've added an additional commit to allow for `NO_LICENSE_KEY` & `NO_LICENCE_KEY`, and if `LICENSE_KEY` is set, will prefill `LICENCE_KEY` thus not breaking compatibility.. hopefully.

**Comment by frebib** [comment]
So I settled on `license` because that's what cfx-server calls the argument. I've squashed your changes in and made it consistent across the board. Thanks for the PR and the kick to fix our laziness! :grin:

File: docker-compose.yml [hunk 1]
```diff
@@ -13,4 +13,4 @@ services:
       - "30120:30120"
       - "30120:30120/udp"
     environment:
-      LICENCE_KEY: "license-key-here"
+      LICENCE_KEY: "licence-key-here"
```
Comment (reecebenson) [inline docker-compose.yml:16]: Should this be `LICENSE_KEY` to match your changes made to the README?

Comment (frebib) [reply docker-compose.yml:16]: Er oops, good catch. Will push a fix.
You're on it :eyes:

**Pull request closed without merging**